src/main/java/org/xerial/snappy/OSInfo.java

```diff
@@ -226,6 +231,10 @@ public static String getArchName() {
         return translateArchNameToFolderName(osArch);
     }
 
+    private static boolean isMusl() {
+        return new File("/lib/ld-musl-x86_64.so.1").exists();
```

Comment:
Let's wrap this block with try .. catch to consume any IOException here. If an exception happens, this should return false.

Reply:
Thanks for the feedback, already added that try catch block :)